### PR TITLE
debug: Output files that are checked

### LINKED ISSUE #2025: Debug output should contain files that get checked

### PR DESCRIPTION
**Pull request by Uran198**
Fixes #2025

### CODE REVIEW

File: coalib/processes/Processing.py [hunk 1]
```diff
@@ -259,6 +259,8 @@ def get_file_dict(filename_list, log_printer):
                                       exception,
                                       log_level=LOG_LEVEL.WARNING)
 
+    for filename in file_dict.keys():
+        log_printer.debug(filename)
```
Comment (SanketDG) [inline coalib/processes/Processing.py]: Don't we want to have a message that these are checked files? Also, we should have one constructed debug message, not several.

This is how it looks right now, 

```
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Tutorial.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Git_Hooks.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Exit_Codes.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Documentation.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Install.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/MAC_Hints.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Review.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Testing_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/coafile.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Tests.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Git_Help.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/General_Dev_Info/git_tutorial_1.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Good_Commits.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Self_Correcting_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/index.rst
[DEBUG][11:48:45] /home/sanket/coala/README.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Newcomers.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Linter_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Codestyle.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Glob_Patterns.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/README.rst
[DEBUG][11:48:45] /home/sanket/coala/CONTRIBUTING.md
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/External_APIs.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Writing_Bears.rst
[DEBUG][11:48:45] Running bear SpaceConsistencyBear...
[DEBUG][11:48:45] Running bear LineLengthBear...
[DEBUG][11:48:45] Running bear SpaceConsistencyBear...
```

File: coalib/processes/Processing.py [hunk 2]
```diff
@@ -259,6 +259,8 @@ def get_file_dict(filename_list, log_printer):
                                       exception,
                                       log_level=LOG_LEVEL.WARNING)
 
+    log_printer.debug("Files that'll be checked:\n" +
```
Comment (SanketDG) [inline coalib/processes/Processing.py]: "that will"

and this should be good to go. :D

**Comment by SanketDG** [comment]
@Uran198 the tag name should be a file, class or package name. A better commit message would be:

```
Processing: show files that will be checked in debug output
```

**Comment by SanketDG** [comment]
ack 3f4ce15 5184ed7

**Comment by sils1297** [comment]
@Uran198 can you make an asciinema?

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**